src/env.c

```diff
@@ -1,6 +1,26 @@
 #include "rlang.h"
 
+
+bool r_is_unbound_value(SEXP x) {
+  return x == R_UnboundValue;
+}
+
 SEXP r_mut_env_parent(SEXP env, SEXP new_parent) {
```

Comment:
e.g.

```c
r_mut_env_parent(mut sexp* env, sexp* new_parent);
```

versus

```c
r_mut_env_parent(sexp* output_env, sexp* new_parent);
```

The latter does seem clearer upon reading, but maybe easier to overlook.